pkg/api/org_users.go

```diff
@@ -124,33 +113,60 @@ func isOrgAdminFolderAdminOrTeamAdmin(c *models.ReqContext) (bool, error) {
 	return isAdminOfTeamsQuery.Result, nil
 }
 
+func (hs *HTTPServer) GetUsers(c *models.ReqContext) (models.SearchOrgUserQueryResult, error) {
+	// take perpage, then if perpage not valid, take limit. If neither perpage nor limit is valid, put default limit 1000
+	perPage := c.QueryInt("perpage")
+	if perPage <= 0 {
+		perPage = c.QueryInt("limit")
+		if perPage <= 0 {
+			perPage = 1000
+		}
+	}
+	page := c.QueryInt("page")
+	if page < 1 {
+		page = 1
+	}
+
+	searchQuery := c.Query("query")
+
+	query := &models.GetOrgUsersQuery{
+		OrgId: c.OrgId,
+		Query: searchQuery,
+		Limit: perPage,
+		Page:  page,
+	}
+	var err error
+	query.Result.Users, err = hs.getOrgUsersHelper(query, c.SignedInUser)
+
+	query.Result.Page = page
+	query.Result.PerPage = perPage
+
+	return query.Result, err
```

Comment:
Nothing is used from `query.Result` but the `Users` field, so best just return that.

```suggestion
	return hs.getOrgUsersHelper(query, c.SignedInUser)
```